fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -280,13 +324,31 @@ case "$BiasCorrection" in
 		;;
 esac
 
-
 ########################################## DO WORK ########################################## 
 
 T1wFolder="$Path"/"$Subject"/"$T1wFolder"
 AtlasSpaceFolder="$Path"/"$Subject"/"$AtlasSpaceFolder"
 ResultsFolder="$AtlasSpaceFolder"/"$ResultsFolder"/"$NameOffMRI"
 
+# set reference and check if external reference (if one is specified) exists 
+
+if [ ! $fMRIReference = "NONE" ] ; then  
+  fMRIReferencePath="$Path"/"$Subject"/"$fMRIReference"
+  log_Msg "Using reference image from ${fMRIReferencePath}"
+  fMRIReferenceImage="$fMRIReferencePath"/"$ScoutName"_gdc
+
+  if [ "$fMRIReferencePath" = "$fMRIFolder" ] ; then
+    log_Err_Abort "Specified BOLD Reference is the same as the current BOLD!"
```

Comment:
Elaborate slightly:
`log_Err_Abort "Specified BOLD reference (--fmriref) is the same as the current BOLD (--fmriname)!"`

Reply:
done